src/plugins/data_source_management/public/components/create_data_source_wizard/components/create_form/create_data_source_form.test.tsx

```diff
@@ -17,7 +17,10 @@ import {
   sigV4AuthMethod,
   usernamePasswordAuthMethod,
 } from '../../../../types';
-import { AuthenticationMethodRegistery } from 'src/plugins/data_source_management/public/auth_registry';
+import {
+  AuthenticationMethod,
+  AuthenticationMethodRegistery,
+} from 'src/plugins/data_source_management/public/auth_registry';
```

Comment:
should we use the relative path instead of absolute path here?

Reply:
Path updated, thanks~